core/src/main/java/com/linecorp/armeria/server/ServerMetrics.java

```diff
@@ -162,11 +162,12 @@ public void bindTo(MeterRegistry meterRegistry) {
         meterRegistry.gauge("armeria.server.pending.requests",
                             ImmutableList.of(Tag.of("protocol", "http2")), pendingHttp2Requests);
         // Active requests
-        meterRegistry.gauge("armeria.server.active.requests", ImmutableList.of(Tag.of("protocol", "http1")),
+        final String activeRequestMeterName = "armeria.server.active.requests.all";
```

Comment:
How about `armeria.server.all.active.requests`? It sounds more natural. I guess people don't use `armeria.server.all` as their prefix? (or we could reject when a user specifies it)

Reply:
Otherwise, I think `active` and `pending` could be set as a `state` tag that would be more in line with `all`.
```
# Get all requests
armeria.server.all.request
# Get all requests with 'active' state
armeria.server.all.requests{state=active}
# Get all requests with 'pending' state
armeria.server.all.request{state=active}
```

Reply:
That also looks good. 👍